docs/source/features_conceptual/message_length.rst

```diff
@@ -0,0 +1,36 @@
+.. _TEMPLATE:
+
+Message Length
+============
+
+High-Level Intuition
+*********************
+Returns the number of words in a message/utterance
+
+Citation
+*********
+NA
+
+Implementation Basics 
+**********************
+
+Returns the number of words in a message/utterance
```

Comment:
We need to say this is basically just splitting on the whitespace, after preprocessing to remove punctuation.